samples/csharp_dotnetcore/70.skills-simple-bot-to-bot/README.md

```diff
@@ -0,0 +1,27 @@
+# SimpleBotToBot (**DRAFT**)
+
+Bot Framework v4 Skills Echo sample.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com), it shows how to use skills from a rootbot.
+
+## Prerequisites
+
+- [.NET Core SDK](https://dotnet.microsoft.com/download) version 2.1
+
+  ```bash
+  # determine dotnet version
+  dotnet --version
+  ```
+
+## Key concepts
+
+- A [SimpleRootBot](SimpleRootBot/README.md) that can consume an echo skill.
+- A [EchoSkillBot](EchoSkillBot/README.md) that handle echoes message activities from the parent bot.
+
+## To try this sample
+- Open the solution and configure it to run both projects (TODO: explain this)
+
+- Create a bot registration in the azure portal for the EchoSkillBot and update [EchoSkillBot/appsettings.json](EchoSkillBot/appsettings.json) with the AppId and password.
```

Comment:
I generally prefer to spell out app ID, or to use it as a literal, `appId`.
```suggestion
- Create a bot registration in the azure portal for the EchoSkillBot and update [EchoSkillBot/appsettings.json](EchoSkillBot/appsettings.json) with the app ID and password.
```

Reply:
Addressed